docs/overview.rst

```diff
@@ -2,40 +2,60 @@ Overview
 ========
 
 ``fmu-dataio`` is a library for exporting data out of FMU workflows. In addition to making
-data export on the current (filename-based) standard easier, it also exports rich metadata
-making it possible to utilize cloud storage (Sumo). Further, centralizing export methods
-like this makes workflows simpler and less prone to break when changes are made.
+data export on the current (filename-based) standard easier, it also creates and attaches
+metadata to the exported data.
+
+In a Python script running somewhere in the FMU workflow, usage of ``fmu-dataio`` looks
+roughly like this (more detailed working examples are provided elsewhere in this documentation):
+
+.. code-block:: python
+
+    from fmu.dataio import ExportData
+
+    df = make_my_data() # how you create a data object is same as before
+    cfg = get_global_config() # how you read global config is same as before
+
+    exp = ExportData( # ExportData takes a number of arguments, these are examples.
+        config=cfg,
+        content="volumes",
+    )
+    exp.export(df) # this is the export.
+
+Although long-term intention is to discontinue this, exported data will still be stored
+to the underlying disk structure on which FMU is running (i.e. /scratch). When storing
+data to disk, ``fmu-dataio`` will store the data file according to the current filename-oriented
+FMU data standard. Next to it, with a corresponding file name, it will store the metadata according
+to the metadata-based FMU data standard.
+
+Example:
+
+.. code-block:: console
+
+    share/results/tables/mytable.csv
+    share/results/tables/.mytable.csv <-- New!
```

Comment:
Would this actually be `.mytable.csv.yml`?

Reply:
Yes! Will fix 👍

Reply:
Will also change "New!" to "metadata"

Reply:
Done